packages/schema-derive/src/generate_api.rs

```diff
@@ -159,10 +159,13 @@ impl Parse for Options {
             }
         };
 
-        let instantiate = map
-            .remove(&parse_quote!(instantiate))
-            .unwrap()
-            .unwrap_type();
+        let instantiate = match map.remove(&parse_quote!(instantiate)) {
+            Some(ty) => {
+                let ty = ty.unwrap_type();
+                quote! {Some(::cosmwasm_schema::schema_for!(#ty))}
+            }
+            None => quote! { None },
+        };
```

Comment:
This codecov message probably memans we have no test with a None instantiate field. Is this something we can easily add?